src/moai-sim/MOAIEaseType.h

```diff
@@ -28,6 +28,21 @@
 	@const	SOFT_EASE_OUT					t ^ 2
 	@const	SOFT_SMOOTH						t ^ 2
 	@const	SOFT_SMOOTH_EASE_OUT			t ^ 2
+	@const	SINE_EASE_IN	
```

Comment:
It would be great if you could add descriptions to your ease types, like for the existing ones.